src/Symfony/Component/String/Resources/functions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String;
+
+function g(string $string = ''): GraphemeString
+{
+    return new GraphemeString($string);
+}
```

Comment:
any thoughts on `function str` for a main entry point?

Reply:
str is abstract, b/u/g are concrete: they explicitly ask for a unit system.

Reply:
yes, i was wondering if `g` is the sane default ... so an alias actually.